sjb/config/test_cases/test_branch_origin_extended_conformance_crio.yml

```diff
@@ -58,6 +58,7 @@ extensions:
                          --inventory sjb/inventory/ \
                          -e deployment_type=origin  \
                          -e openshift_use_crio=True   \
+                         -e openshift_crio_systemcontainer_image_registry_override=docker.io/gscrivano \
```

Comment:
Should this also specify the name `cri-o-centos`|`cri-o-fedora`? This PR, afaict, is now pulling `docker.io/gscrivano/cri-o:latest` which doesn't exist. @giuseppe @ashcrow

Reply:
@runcom Yes, you're correct. The other option which may be quicker and easier (as either would be temporary anyway) would be to tag a ``cri-o`` image in @giuseppe's docker.io.

Reply:
@ashcrow do we need to change this in openshift-ansible?  I think it is reasonable to override the entire image not just the registry, what do you think if we drop the `openshift_crio_systemcontainer_image_registry_override` logic in favour of `openshift_crio_systemcontainer_image_override` and do the same for the docker system container?

Reply:
@giuseppe I don't think we need to but we could. If you and @runcom both think it would be easier to have a full override I'll take care of modifying and getting a PR up for it today. I've had enough ☕️ already to be in code mode 😹

Reply:
I've tagged 'docker.io/gscrivano/cri-o' for now.

Reply:
I'll make the change but this can merge.